server/routes/employer-sites.js

```diff
@@ -87,8 +89,38 @@ router.get(
   ],
   asyncMiddleware(async (req, res) => {
     const { hcapUserInfo: user } = req;
+    const result = await getAllSites();
 
-    const result = await getSitesForUser(user);
+    logger.info({
+      action: 'employer-sites_get-all',
+      performed_by: {
+        username: user.username,
+        id: user.id,
+      },
+      sites_accessed: result.map((site) => site.siteId),
+    });
+    return res.json({ data: result });
+  })
+);
+
+// Read: Get All User Sites
+router.get(
+  '/user',
```

Comment:
Why not sites/region?

I guess that wouldn't let us use multiple sites but I don't think that's a real use case is it?

Reply:
If we do, on the frontend we'll have to differentiate the user on the Sites component. As in

if user.isHA, request sites/region
else if user.isMoH, request sites

I'm leaning towards not in this case. Don't love the query flag, but there's a little annoyance no matter which way we try it

Reply:
Ah fair enough 👍